src/prasad/BankTransactions.java

```diff
@@ -0,0 +1,199 @@
+package prasad;
+
+import java.util.Scanner;
+
+public class BankTransactions {
+	float icicibalance;
+	int icicicreditcount;
+	int icicidebitcount;
+	float hdfcbalance;
+	int hdfccreditcount;
+	int hdfcdebitcount;
+	float bobbalance;
+	int bobcreditcount;
+	int bobdebitcount;
+	float sbibalance;
+	int sbicreditcount;
+	int sbidebitcount;
+	int totaldebitcount;
+	int totalcreditcount;
+	
+	void credit(String bank,float amount) {
+		switch(bank.toUpperCase()) {
+		case "ICICI":
+			icicibalance = icicibalance+amount;
+			icicicreditcount++;
+			totalcreditcount++;
+			break;
+		case "HDFC":
+			hdfcbalance = hdfcbalance+amount;
+			hdfccreditcount++;
+			totalcreditcount++;
+			break;
+		case "BOB":
+			bobbalance = bobbalance+amount;
+			bobcreditcount++;
+			totalcreditcount++;
+			break;
+		case "SBI":
+			sbibalance = sbibalance+amount;
+			sbicreditcount++;
+			totalcreditcount++;
+			break;
+		default:
+			System.out.println("Transaction failed: Invalid bank name.");
+		}
+	}
+	void debit(String bank,float amount) {
+		switch(bank.toUpperCase()) {
+		case "ICICI":
+			if(amount<icicibalance) {
+				icicibalance=icicibalance-amount;
+				icicidebitcount++;	
+				totaldebitcount++;
+			}else 
+				System.out.println("Transaction failed: Insufficient balance.");
+			break;
+		case "HDFC":
+			if(amount<hdfcbalance) {
+				hdfcbalance=hdfcbalance-amount;
+				hdfcdebitcount++;	
+				totaldebitcount++;
+			}else 
+				System.out.println("Transaction failed: Insufficient balance.");
+			break;
+		case "BOB":
+			if(amount<bobbalance) {
+				bobbalance=bobbalance-amount;
+				bobdebitcount++;	
+				totaldebitcount++;
+			}else 
+				System.out.println("Transaction failed: Insufficient balance.");
+			break;
+		case "SBI":
+			if(amount<sbibalance) {
+				sbibalance=sbibalance-amount;
+				sbidebitcount++;	
+				totaldebitcount++;
+			}else 
+				System.out.println("Transaction failed: Insufficient balance.");
+			break;
+		default:
+			System.out.println("Transaction failed: Invalid bank name.");
+		}
+	}
+	float totalbalance() {
+		float totalbalance=0;
+		totalbalance = icicibalance+sbibalance+hdfcbalance+bobbalance;
```

Comment:
You can write this in single line right
float totalbalance = icicibalance+sbibalance+hdfcbalance+bobbalance;